packages/react-router-dom/index.tsx

```diff
@@ -1072,23 +1082,6 @@ export interface FormProps extends FetcherFormProps {
   unstable_viewTransition?: boolean;
 }
 
-/**
- * A `@remix-run/router`-aware `<form>`. It behaves like a normal form except
- * that the interaction with the server is with `fetch` instead of new document
- * requests, allowing components to add nicer UX to the page as the form is
- * submitted and returns with data.
- */
-export const Form = React.forwardRef<HTMLFormElement, FormProps>(
-  (props, ref) => {
-    let submit = useSubmit();
```

Comment:
Now that we're back to one `useSubmit` method, we don't need this abstraction anymore - a fetcher can just use `<Form navigate={false} fetcherKey={fetcherKey}>` internally 😍